_component_design/radio-buttons.md

```diff
@@ -7,157 +7,140 @@ usage: >
-    </div>
-    <div class="ui message subtext">
-      Here is some additional help text to explain these choices.
-    </div>
-  </div>
 </div>
 
-## Disabled
+### States
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Overview
+
+A title should always be located above the radio buttons. More context can be provided under the title (subtitle) but it is optional (shown here as “Select an option” on left). If subtitle is not needed, use specs in the design on the right.
+
+Best practice form design would not allow a radio button to ever exist in error state, so try to avoid it at all cost.
+
+
+{% endcolumn %}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Radio button states"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/radio-buttons/radio-states.svg)
+{% endfigure %}
+{% endcolumn %}
 
-<div class="ui form">
-  <div class="field">
-    <label>Select payment type</label>
-    <div class="ui buttons single">
-      <button class="ui button ds-btn-med-sec disabled active">Credit / Debit</button><button class="ui button ds-btn-med-sec disabled">Cash</button><button class="ui button ds-btn-med-sec disabled">Check</button>
-    </div>
-  </div>
 </div>
 
-## Error
-
-<div class="ui form error">
-  <div class="field error">
-    <label class="required">Select potential maintenance windows</label>
-    <div class="ui buttons single">
-      <button class="ui button ds-btn-med-sec">12<span class="smCaps"> AM</span> &ndash; 4<span class="smCaps"> AM</span></button><button class="ui button ds-btn-med-sec error active">4<span class="smCaps"> AM</span> &ndash; 8<span class="smCaps"> AM</span></button><button class="ui button ds-btn-med-sec">8<span class="smCaps"> AM</span> &ndash; 12<span class="smCaps"> PM</span></button><button class="ui button ds-btn-med-sec">12<span class="smCaps"> PM</span> &ndash; 4<span class="smCaps"> PM</span></button><button class="ui button ds-btn-med-sec">4<span class="smCaps"> PM</span> &ndash; 8<span class="smCaps"> PM</span></button><button class="ui button ds-btn-med-sec">8<span class="smCaps"> PM</span> &ndash; 12<span class="smCaps"> AM</span></button>
-    </div>
-    <div class="ui error message">
-      There is an error with the response to this question.
-    </div>
-  </div>
+
+
+<div class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Error state - expanded
+
+The question with the error receives an asterisk. The asterisk position floats to the left, in the margin, not in vertical alignment with the text. The section label should remain in its natural vertical alignment. 
+
+Only the option with the error receives the error radio button.
+
+Let the error message wrap to fixed upper and lower bounds, so as to adhere to 40-80 characters per line.
```

Comment:
...bounds, with 40 to 80 characters per line.

Reply:
updated.